moabb/datasets/gigadb.py

```diff
@@ -127,4 +127,4 @@ def data_path(
             raise (ValueError("Invalid subject number"))
 
         url = "{:s}s{:02d}.mat".format(GIGA_URL, subject)
-        return dl.data_path(url, "GIGADB", path, force_update, update_path, verbose)
+        return dl.data_dl(url, "GIGADB", path, force_update, update_path, verbose)
```

Comment:
this could be somewhat confusing here, as the function `data_path` returns the call to `dl.data_dl`, but I dont know a better solution while `dl.data_path` is deprecating

Reply:
Agree, this is confusing at first. This is mostly to kept a naming convention that is consistent with MNE. Each MNE dataset define its [own `data_path` method](https://mne.tools/stable/generated/mne.datasets.sample.data_path.html). Now that we have our own getter `data_dl`, this should be more readable.